setup.py

```diff
@@ -52,15 +52,6 @@
 
 """
 
-install_requires = [
-    "trio >= 0.15.0",
-    "outcome",
-    "sniffio",
-]
-if sys.version_info < (3, 7):
-    install_requires.append("contextvars >= 2.1")
-    install_requires.append("async_generator >= 1.6")
-
 setup(
```

Comment:
Perhaps we could eliminate `setup.py` entirely since this project doesn't have any C extensions?

Reply:
I don't know how to do that and keep a single source of truth about the package version. In `setup.py` we load the version from `trio_asyncio/_version.py`.

Reply:
That can be done (instructions [here](https://setuptools.pypa.io/en/latest/userguide/pyproject_config.html#dynamic-metadata)).

Reply:
Good to know. I think I'll leave that for another PR though; this one is currently blocking a release and I don't want to complicate it further.